server/util/filter/filter.go

```diff
@@ -29,6 +29,8 @@ func executionMetricToDbField(m stat_filter.ExecutionMetricType) (string, error)
 		return "file_download_size_bytes", nil
 	case stat_filter.ExecutionMetricType_OUTPUT_UPLOAD_SIZE_EXECUTION_METRIC:
 		return "file_upload_size_bytes", nil
+	case stat_filter.ExecutionMetricType_EXECUTION_WALL_TIME_EXECUTION_METRIC:
+		return "IF(worker_completed_timestamp_usec < queued_timestamp_usec, 0, (worker_completed_timestamp_usec - queued_timestamp_usec))", nil
```

Comment:
i included queue time as part of the overall number because (most of the time) i think users are wondering "what's slow", and queueing can be part of that.  we also have measurements for each block of time (queue, input dl, ...), so users can kinda check out which piece is actually slow once they've narrowed things down a bit.